runtime/src/iree/hal/drivers/hip/pending_queue_actions.c

```diff
@@ -524,56 +505,61 @@ iree_hal_hip_pending_queue_actions_cast(iree_hal_resource_t* base_value) {
   return (iree_hal_hip_pending_queue_actions_t*)base_value;
 }
 
-static bool iree_hal_hip_worker_committed_exiting(
-    iree_hal_hip_working_area_t* working_area);
+static void iree_hal_hip_pending_queue_actions_notify_worker_thread(
+    iree_hal_hip_working_area_t* working_area) {
+  iree_atomic_store_int32(&working_area->worker_state,
+                          IREE_HAL_HIP_WORKER_STATE_WORKLOAD_PENDING,
+                          iree_memory_order_release);
+  iree_notification_post(&working_area->state_notification, IREE_ALL_WAITERS);
+}
+
+static void iree_hal_hip_pending_queue_actions_notify_completion_thread(
+    iree_hal_hip_completion_area_t* completion_area) {
+  iree_atomic_store_int32(&completion_area->worker_state,
+                          IREE_HAL_HIP_WORKER_STATE_WORKLOAD_PENDING,
+                          iree_memory_order_release);
+  iree_notification_post(&completion_area->state_notification,
+                         IREE_ALL_WAITERS);
+}
+
+// Notify work and completion threads that there is work.
```

Comment:
style: https://google.github.io/styleguide/cppguide.html#Function_Comments
(here "// Notifies worker and completion threads that there is work available to process.")

Reply:
Done.